src/lib/features/segment/segment-store.ts

```diff
@@ -123,17 +124,15 @@ export default class SegmentStore implements ISegmentStore {
 
     private async getAllWithoutChangeRequestUsageData(): Promise<ISegment[]> {
         const rows: ISegmentRow[] = await this.db
-            .select(
-                this.prefixColumns(),
-                'used_in_projects',
-                'used_in_features',
-            )
-            .countDistinct(
-                `${T.featureStrategies}.project_name AS used_in_projects`,
-            )
-            .countDistinct(
-                `${T.featureStrategies}.feature_name AS used_in_features`,
-            )
+            .select([
+                ...this.prefixColumns(),
+                this.db.raw(
```

Comment:
The COUNT functions are now wrapped with CASE statements. These CASE statements ensure that only non-archived features (where features.archived_at is NULL) are counted. If a feature is archived, the CASE returns NULL, which does not affect the count, effectively making it 0